app/controllers/admin/base_items_controller.rb

```diff
@@ -38,9 +38,12 @@ def show
     @items = @base_item.items
   end
 
+  # TODO there are no buttons on the view to call this method, consider removing?
```

Comment:
We're going to be removing the current approach to base items at some point in the future. I think for now it's safe to just leave this method alone if nothing calls it.

Reply:
added a note for this in PR #4665 [a74dcc9](https://github.com/rubyforgood/human-essentials/pull/4665/commits/a74dcc9b2d89e90b410404aef99ee07c6af694e4)